.github/workflows/int_test.yml

```diff
@@ -236,14 +249,14 @@ jobs:
           cluster-name: ${{ env.CLUSTER_NAME }}
           resource-group: ${{ env.RESOURCE_GROUP }}
           custom-locations-oid: ${{ env.CUSTOM_LOCATIONS_OID }}
-      - name: "Tox test environment setup for init"
-        run: |
-          python -m pip install tox
-          tox r -vv -e python-init-int --notest
-      - name: "Tox test environment setup for integration tests"
-        if: ${{ matrix.feature == 'default' && !inputs.use-container }}
-        run: |
-          tox r -vv -e "python-all-int" --notest
+          enable-workload-identity: ${{ matrix.feature == 'trustbundle' }}
+      - name: "Az CLI login"
+        if: ${{ matrix.feature == 'trustbundle' && !inputs.use-container }}
```

Comment:
Do we need this `if` here?

I think since ARC might still somehow take too long, I'm ok with another login step here regardless to ensure the token doesn't timeout before the tests start.

Reply:
well originally, i had it as an if cause I def needed it for the trustbundle part but if we may as well have it for everything, will remove the if